README.md

```diff
@@ -4,6 +4,10 @@ BattETL is a well-tested and an enterprise-ready python module for **E**xtractin
 
 <img src="./images/BattETL_Simple_System.svg">
 
+- [Installation & setup video](https://www.youtube.com/watch?v=tmudLhrj9xE)
```

Comment:
@tanevitt can we refactor the README a bit, so these links are under the complete navigation tree? 

A sub-section like "Videos" perhaps?

Reply:
Sure. Do you want me to remove references to them elsewhere?

Reply:
I mean, move the references.

Reply:
Done. Re-review requested.